src/main/java/org/finance_price_service/entrypoint/PricesController.java

```diff
@@ -0,0 +1,27 @@
+package org.finance_price_service.entrypoint;
+  import org.finance_price_service.domain.PricesSet;
+  import org.finance_price_service.service.QueryLogicService;
+  import org.springframework.beans.factory.annotation.Autowired;
+  import org.springframework.web.bind.annotation.RequestMapping;
+  import org.springframework.web.bind.annotation.RequestParam;
+  import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * Calls QueryLogicService for dealing with requests
+ */
+@RestController
+public class PricesController {
+  @Autowired
+  private QueryLogicService logic;
+  @RequestMapping("/query")
+  public PricesSet response(@RequestParam(value = "symbol") String symbol, @RequestParam(value = "days", defaultValue = "0") int days)
```

Comment:
add a javadoc to each of these methods.  Because they are the entrypoints for users they are important to have javadocs for. A general rule of thumb is that you should add javadocs for any public method